src/listmode_buildblock/LmToProjData.cxx

```diff
@@ -399,16 +399,16 @@ get_bin_from_event(Bin& bin, const ListEvent& event) const
 
 
     // do_normalisation
-#ifndef FRAME_BASED_DT_CORR
-     const double start_time = current_time;
-     const double end_time = current_time;
-#else
-     const double start_time = frame_defs.get_start_time(current_frame_num);
-     const double end_time =frame_defs.get_end_time(current_frame_num);
-#endif
+//#ifndef FRAME_BASED_DT_CORR
+//     const double start_time = current_time;
+//     const double end_time = current_time;
+//#else
+//     const double start_time = frame_defs.get_start_time(current_frame_num);
+//     const double end_time =frame_defs.get_end_time(current_frame_num);
+//#endif
```

Comment:
this is the bit that needs some thought here.

Reply:
yes I was not sure about this indeed.

Reply:
this will be resolved once we fix #794